sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -88,7 +90,7 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
-    std::chrono::milliseconds RetryDelay = std::chrono::seconds(4);
+    std::chrono::milliseconds RetryDelay = std::chrono::milliseconds(800);
```

Comment:
The C++ SDK is out of compliance with all the Azure SDKs. 

Here's the relevant text:
```
YOU SHOULD implement the default policy with 3 retries with a 0.8s exponential (plus jitter) delay between tries with a maximum delay between tries of 60 seconds. 
```

There was a recent SDK Arch team discussion and this was part of the fallout.

Reply:
It might be worth calling this out in the changelog. We can discuss a bit more, offline, on how impactful (if at all) such a change might be to customers.